manifests/profiles/demo.yaml

```diff
@@ -12,6 +12,11 @@ spec:
       skywalking:
         service: tracing.istio-system.svc.cluster.local
         port: 11800
+    - name: otel
+      opentelemetry:
+        port: 4317
+        service: opentelemetry-collector.istio-system.svc.cluster.local
```

Comment:
If this is for a demo - at least for the new stuff it would be great to not show examples with collectors and other telemetry components in istio-system. 

We have the citadel root private key there and the control plane - we recommend as best practice to keep even ingress in separate namespaces. OTel collector should be in whatever namespace the otel examples and default install uses.